cli/cliflags/names.go

```diff
@@ -33,6 +33,7 @@ const (
 	PasswordName          = "password"
 	PortName              = "port"
 	HTTPPortName          = "http-port"
+	HTTPIPAddrName        = "http-addr"
```

Comment:
Actually, a bigger nit: let's stick to the same naming scheme as `{host, port}`, where we now have `{http-host, http-port}`.